.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,14 +4,20 @@ This file outlines a list of common things that should be addressed when opening
 
 ## Please fill out the following before requesting review on this PR
 
-**Description of Change:** _TODO_
+### Description
+*Give a high-level desciprion of the changes in this PR*
 
-Closes #_TODO: Issue number here (this automatically closes the issue when this PR is merged. To add more issues, just copy and fill out this line!)_
+### Testing Done
+*Outline any testing that was done for these changes. This could be unit tests, integration tests, etc.*
 
-### Checklist
+### Resolved Issues
+*Link any issues that this PR resolved. Eg `resolves #1, #2, and #5` (note that they MUST be specified like this so Github can automatically close them then this PR merges)*
+
+### Review Checklist
 *(Please check every item to indicate your code complies with it (by changing `[ ]`->`[x]`). This will hopefully save both you and the reviewer(s) a lot of time!)*
-- [ ] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in `src/sample_package` for examples.
-- [ ] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `src/sample_package`
+***It is the reviewers responsibility to also make sure every item here has been covered***
+- [ ] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in the `thunderbots/software/geom` folder for examples.
+- [ ] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `thunderbots/software/geom` 
```

Comment:
Same thing here, not sure if you meant to refer to the thunderbots repo.

Reply:
Whoops, good catch! Fixed to point to our examples.